test/python/common.py

```diff
@@ -91,3 +91,29 @@ def generate_random_numbers(self,
         if datatype == "DTYPE_INT":
             return [random.randint(start, end) for i in range(vector_length)]
         return [random.uniform(start, end) for i in range(vector_length)]
+
+    # returns true if the the env variable is set
+    def is_env_variable_set(self, env_var):
+        return env_var in os.environ
+
+    # sets the env variable
+    def set_env_variable(self, env_var, env_var_val):
```

Comment:
Is this only for setting the backend env variable? If yes, then please change the name to project that.
If not, then please change the `print message`

Reply:
This is for setting any env variable.